### PR TITLE
[Improvement][API] simplify with stream

### PR DESCRIPTION
**Pull request by SunJiFengPlus**
simplify for loop with stream

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java [hunk 1]
```diff
@@ -232,25 +232,14 @@
...
-            if (sb.length() > 0) {
-                sb.append(",");
-            }
-            sb.append(i);
-        }
-        return sb.toString();
+        return processData.getTasks()
+                .stream()
+                .map(taskNode -> TaskParametersUtils.getParameters(taskNode.getType(), taskNode.getParams()))
+                .filter(Objects::nonNull)
```
Comment (Technoboy-) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java:238]: Filter here is for - AbstractParameters.  not for params.getResourceFilesList(). please check.

**Comment by dailidong** [comment]
good job， could you add unit test for this method?

**Comment by SunJiFengPlus** [comment]
> good job， could you add unit test for this method?

completely fine, i will finish this before this weekend

**Review by dailidong — APPROVED** [review]
+1
good job

thanks for your first contribution, please contact me( wechat(510570367) or mail(dailidong66@gmail.com)) , when added, please tell me your github id, thx

**Pull request merged**